src/test/java/io/github/alekseykn/imnorm/TransactionTest.java

```diff
@@ -0,0 +1,157 @@
+package io.github.alekseykn.imnorm;
+
+import io.github.alekseykn.imnorm.exceptions.OtherTransactionRedactCurrentClusterException;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import support.dto.Dto;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class TransactionTest {
+    private static Repository<Dto> repository;
+
+    @BeforeAll
+    static void init() {
```

Comment:
it is not necessary to take it out in the method

Reply:
Done